pytorch_lightning/loggers/wandb.py

```diff
@@ -132,11 +132,8 @@ def log_metrics(self, metrics: Dict[str, float], step: Optional[int] = None) ->
 
     @property
     def name(self) -> str:
-        # don't create an experiment if we don't have one
-        name = self._experiment.project_name() if self._experiment else None
-        return name
+        return self.experiment.project_name()
```

Comment:
what about the "don't create an experiment if we don't have one"

Reply:
https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/loggers/wandb.py#L98
self.experiment already does this